www/js/index.js

```diff
@@ -22,38 +22,30 @@ function escKey() {
   } */
 }
 
-function waheguruSlide() {
-  // waheguru slide shortcut
-  global.controller.sendText('vwihgurU', true);
-}
-function moolMantraSlide() {
-  // ik oankar slide shortcut
-  global.controller.sendText(
-    '<> siq nwmu krqw purKu inrBau inrvYru Akwl mUriq AjUnI sYBM gur pRswid ]',
-    true,
-  );
-}
-function emptySlide() {
-  // show Empty Slide
-  global.controller.sendText('');
-}
-function anandSahibBhog() {
-  // anand sahib (6 pauri) shortcut
-  global.core.search.loadCeremony(3).catch(error => {
-    analytics.trackEvent('ceremonyFailed', 3, error);
-  });
-}
-function helpGuideShortcut() {
-  // help window
-  main.openSecondaryWindow('helpWindow');
-}
-function legendShortcut() {
-  // shortcut legend window
-  main.openSecondaryWindow('shortcutLegend');
-}
-function searchBarShortcut() {
-  search.$search.focus();
-}
+const slideShortcuts = {
+  waheguru: () => global.controller.sendText('vwihgurU', true),
+  empty: () => global.controller.sendText(' '),
+  moolMantra: () => {
+    global.controller.sendText(
+      '<> siq nwmu krqw purKu inrBau inrvYru Akwl mUriq AjUnI sYBM gur pRswid ]',
```

Comment:
pulls these strings out in to a file just for constants and require them. They are probably used in multiple places so its better to keep it DRY(**D**ont**R**epeat**Y**ourself).

Reply:
Would u recommend a slideStrings.json or something because some of these are using in things like the toolbar

Reply:
doesn't have to be a `.json` file.  `slideStrings.js` and in it do
```
export const slideString = {
 moolMantar: <> siq nwmu krqw purKu inrBau inrvYru Akwl mUriq AjUnI sYBM gur pRswid ]
 ...
}
```

and then use it in index.js like 
```
const slideString = require('./slideStrings');

...
global.controller.sendText(
      slideString.moolMantar,
      true,
    );
...
```

Reply:
This seems to break everything. It doesn't understand the export for some reason

Reply:
Im just gonna do a json for now